src/main/scala/dpla/ingestion3/mappers/providers/FlMapping.scala

```diff
@@ -0,0 +1,127 @@
+
+
+class FlMapping extends JsonMapping with JsonExtractor with IngestMessageTemplates {
+
+  val formatBlockList: Set[String] = ExtentIdentificationList.termList
+
+  // ID minting functions
+  override def useProviderName: Boolean = true
+
+  override def getProviderName: String = "florida"
+
+  override def originalId(implicit data: Document[JValue]): ZeroToOne[String] =
+    extractStrings(unwrap(data) \ "isShownAt").headOption
+
+  // OreAggregation
+  override def dataProvider(data: Document[JValue]): ZeroToMany[EdmAgent] =
+    extractStrings(unwrap(data) \ "dataProvider").map(nameOnlyAgent)
+
+  override def dplaUri(data: Document[JValue]): ZeroToOne[URI] = mintDplaItemUri(data)
+
+  override def edmRights(data: Document[json4s.JValue]): ZeroToMany[URI] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "rights" \ "@id")
+      .filter(_.nonEmpty) // FIXME filtering non-empty values should be a standard edmRights normalization
+      .map(URI)
+
+  override def intermediateProvider(data: Document[JValue]): ZeroToOne[EdmAgent] =
+    extractString(unwrap(data) \ "intermediateProvider")
+      .map(nameOnlyAgent)
+
+  override def isShownAt(data: Document[JValue]): ZeroToMany[EdmWebResource] =
+    extractStrings(unwrap(data) \ "isShownAt")
+      .map(stringOnlyWebResource)
+
+  override def preview(data: Document[JValue]): ZeroToMany[EdmWebResource] =
+    extractStrings(unwrap(data) \ "preview").map(stringOnlyWebResource)
+
+  override def originalRecord(data: Document[JValue]): ExactlyOne[String] =
+    Utils.formatJson(data)
+
+  override def provider(data: Document[JValue]): ExactlyOne[EdmAgent] = agent
+
+  override def sidecar(data: Document[JValue]): JValue =
+    ("prehashId", buildProviderBaseId()(data)) ~ ("dplaId", mintDplaId(data))
+
+  // SourceResource
+  override def alternateTitle(data: Document[JValue]): ZeroToMany[String] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "alternative")
+
+  override def contributor(data: Document[JValue]): ZeroToMany[EdmAgent] =
+    extractStrings(unwrap(data)  \ "sourceResource" \ "contributor" \ "name")
+      .map(nameOnlyAgent)
+
+  override def creator(data: Document[JValue]): ZeroToMany[EdmAgent] =
+    extractStrings(unwrap(data)  \ "sourceResource" \ "creator" \ "name")
+      .map(nameOnlyAgent)
+
+  override def description(data: Document[JValue]): ZeroToMany[String] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "description")
+
+  override def extent(data: Document[JValue]): ZeroToMany[String] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "extent")
+
+  override def format(data: Document[JValue]): ZeroToMany[String] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "format") ++
+      extractStrings(unwrap(data) \ "sourceResource" \ "genre" \ "name")
+
+  override def genre(data: Document[JValue]): ZeroToMany[SkosConcept] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "genre" \ "name")
+      .map(nameOnlyConcept)
```

Comment:
Just sanity checking that Florida's `sourceResource.genre.name` is supposed to be mapped to both `format` and `genre` fields.

Reply:
That is what the mapping says. I think there may be some revisions once we inspect values in CQA but this is following the mapping to the letter.